msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -422,7 +422,24 @@ else { /* handle case where capability isn't supported */ }
 
 ```
 
-#### Generate a deep link to a call
+## Generate a deep link to a meeting
```

Comment:
Share content to a meeting

Reply:
We should also reflect the feature stage - example if its in dev preview, lets add that as a note.